src/wp-content/themes/twentyten/loop.php

```diff
@@ -20,10 +20,25 @@
 ?>
 
 <?php // Display navigation to next/previous pages when applicable. ?>
-<?php if ( $wp_query->max_num_pages > 1 ) : ?>
+<?php
+if ( $wp_query->max_num_pages > 1 ) :
+	$order   = get_query_var( 'order', 'DESC' );
+	$is_desc = ( 'DESC' === $order );
```

Comment:
Nit:

```suggestion
	$is_desc = ( get_query_var( 'order', 'DESC' ) === $order );
```

Reply:
I edited this change so it only combines the two lines into one. The condition should take the query variable, when provided, instead of the global `$order` value.

Did you mean something like this instead?
`$is_desc = ( 'DESC' === get_query_var( 'order', $order ) );`